.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v0.3.5
+          triggered-ref: LLT-3683-GH-natlab
```

Comment:
Adding a message in order not to forget to update this with tag once libtelio-build is merged